examples/patbench.cc

```diff
@@ -461,6 +463,7 @@ class Sigdata {
         vector<const char *> cstrPatterns;
         cstrPatterns.reserve(patterns.size());
         for (const auto &pattern : patterns) {
+            // cppcheck-suppress useStlAlgorithm
```

Comment:
this might be a std::copy with std::back_inserter, no?

Reply:
with a transform, but yes, done.